src/release.sh

```diff
@@ -14,12 +14,16 @@ NPM_INFO="undefined"
 # instead of using -v use -n to check for an empty strings
 # -v is not working well on bash 3.2 on osx
 PATTERNFLY_DEPS="undefined"
+FEC_DEPS="undefined"
 if [[ -f package-lock.json ]] || [[ -f yarn.lock ]];
 then
   LINES=`npm list --silent --depth=0 --production | grep @patternfly -i | sed -E "s/^(.{0})(.{4})/\1/" | tr "\n" "," | sed -E "s/,/\",\"/g"` 
   PATTERNFLY_DEPS="[\"${LINES%???}\"]"
+  LINES=`npm list --silent --depth=0 --production | grep @redhat-cloud-services/frontend -i | sed -E "s/^(.{0})(.{4})/\1/" | tr "\n" "," | sed -E "s/,/\",\"/g"`
```

Comment:
@john-dupuy do we want to include just information about frontend dependencies? Anything from https://github.com/RedHatInsights/frontend-components/tree/master/packages we can also include information about APIs from https://github.com/RedHatInsights/javascript-clients/tree/master/packages

Reply:
@karelhala won't this command already get the stuff from https://github.com/RedHatInsights/frontend-components/tree/master/packages? 

The information about the APIs would be good to include. Thanks for the suggestion.

Reply:
OK, we will add everything from `@redhat-cloud-services` scope.